release-please-config.json

```diff
@@ -10,7 +9,7 @@
     "packages/sdk-node": {},
     "packages/sdk": {},
     ".": {
-      "release-as": "1.1.0"
+      "release-as": ""
```

Comment:
this should be the normal value for the root package - it's just that last time the python "breaking change" was going to bump to 2.0.0 which didn't seem quite right.